src/main/java/com/ericsson/ei/handlers/RmqHandler.java

```diff
@@ -108,9 +123,17 @@
     @Setter
     @Value("${rabbitmq.consumerName}")
     private String consumerName;
+    
+    @Getter
+    @Value("${spring.data.mongodb.database}")
+    private String bindingKeysDataBaseName;
```

Comment:
I guess its same database for whole EI instance, I think its better to name variable just "dataBaseName", so we avoid confusing it with different databse.
I guess we want to achieve one database per EI instance, if several EI instances using same MongoDb instance.
So we have following structure:
EIInstanceDb1:
   - SubscriptionCollection
   - AggregatedObjectCollection
   - MatchedSubscriptionCollection
   - BindingKeyCollection
   - and so on, migh have missed a collection that EI uses

EIInstanceDb2:
   - SubscriptionCollection
   - AggregatedObjectCollection
   - MatchedSubscriptionCollection
   - BindingKeyCollection
   - and so on

EIInstanceDb3:
   - SubscriptionCollection
   - AggregatedObjectCollection
   - MatchedSubscriptionCollection
   - BindingKeyCollection
   - and so on

I guess the BindingKey Collection name can be hardcoded in a Constant library, so we have the name string defined in one place. I guess we have this for other collection names already.